specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/dedicatedHost.json

```diff
@@ -742,6 +742,9 @@
           "200": {
             "description": "OK"
           },
+          "202": {
```

Comment:
was this added for the modelvalidation issue? if so, this does not need to be here. ModelValidations are for example files. Adding a new response existing operation is causing the breaking change. Please remove